tests/io/test_data_catalog.py

```diff
@@ -373,6 +380,20 @@ def test_config_relative_import(self, sane_config):
         with pytest.raises(DataSetError, match=re.escape(pattern)):
             DataCatalog.from_config(**sane_config)
 
+    def test_config_import_kedro_datasets(self, sane_config, mocker):
+        """Test kedro.extras.datasets default path to the dataset class"""
+        # Spy _load_obj because kedro_datasets is installed
+        class DummyMock:
```

Comment:
Could this be a function instead of a class? That may also make the test clearer. I checked the `pytest-mock` docs, it's not obvious if it can be in this case.

Reply:
This is a good question. This isn't possible, the spy need to work with a class and method. I will add comments on it.